lib/statsd/instrument/assertions.rb

```diff
@@ -66,4 +67,8 @@ def assert_statsd_call(metric_type, metric_name, options = {}, &block)
       metric
     end
   end
+
+  def numeric?(object)
+    true if Float(object) rescue false
```

Comment:
you're right, I wanted to catch numbers wrapped in strings but thats totally unnecessary.